src/overlay/PeerManager.cpp

```diff
@@ -326,30 +326,25 @@ PeerManager::update(PeerRecord& peer, TypeUpdate type)
     {
     case TypeUpdate::SET_OUTBOUND:
     {
-        peer.mType = static_cast<int>(PeerType::OUTBOUND);
+        if (peer.mType == static_cast<int>(PeerType::INBOUND))
```

Comment:
`SET_OUTBOUND` and `SET_PREFERRED` should be called `ENSURE_OUTBOUND` and `ENSURE_NOT_PREFERRED`

Reply:
Done